File: python_programming/practices/old_enough.py
#JS old enough assignment

age = int(input("Please tell me your age:\n"))

if age >= 18:
    print("You are old enough to vote!")
elif age >=16:
    print("You are old enough to drive!")
elif age == 15:
    print("You are old enough to get a drivers permit.")
elif age >= 4:
    print("You are able to go to school")
else:
    print("You are too young to do anything")

File: c_programming/notes/practices/old_enough.c
// JS 6th Old enough

#include <stdio.h>
#include <string.h>

int main(void){
    int age;
    printf("How old are you:\n");
    scanf("%d", &age);

    if(age >=18){
        printf("You are old enough to vote");
    }else if(age == 16){
        printf("You are old enough to drive");
    }else if(age ==15){
        printf("You are old enough to get your permit.");
    }else if(age >= 5){
        printf("You can go to school");
    }else{
        printf("You arent old enough for anything here!!\n");
    }
    





    return 0;
}